SofaKernel/modules/SofaSimpleFem/src/SofaSimpleFem/HexahedronFEMForceField.inl

```diff
@@ -746,12 +746,13 @@ void HexahedronFEMForceField<DataTypes>::initSmall(int i, const Element &elem)
     for(int w=0; w<8; ++w)
         _rotatedInitialElements[i][w] =  _rotations[i]*_initialPoints.getValue()[elem[w]];
 
-    if( _elementStiffnesses.getValue().size() <= (unsigned)i )
+    auto& stiffnesses = *sofa::helper::getWriteAccessor(_elementStiffnesses);
```

Comment:
Should it be getWriteOnlyAccessor?
```
The WriteOnlyAccessor provides an access to the Data without triggering an engine update.
This should be the prefered writeAccessor for most of the cases as it avoids uncessary Data updates.
@warning read access to the Data is NOT up-to-date
```
seems to be the good case here

Reply:
That's a good remark! I did not know exactly the differences